DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/AdminFieldsControllerTests.cs

```diff
@@ -190,6 +195,76 @@ public void AdminFieldBulkPost_updates_temp_data_and_redirects_to_edit()
             }
         }
 
+        [Test]
+        public void RemoveAdminField_removes_admin_field_with_no_user_answers()
+        {
+            using var transaction = new TransactionScope();

```

Comment:
Do we need this when mocking the service?

Reply:
Oops no, removed!